db/migrate/20180603184000_enable_pgcrypto_extension.rb

```diff
@@ -0,0 +1,5 @@
+class EnablePgcryptoExtension < ActiveRecord::Migration[5.1]
+  def change
+    enable_extension 'pgcrypto'
```

Comment:
Should we perhaps use https://www.postgresql.org/docs/9.6/static/uuid-ossp.html? It seems more relevant than `pgcrypto`, which has more crypto stuff.

Reply:
We discussed this; ActiveRecord has moved to `pgcrypto` in PG >= 9.4
http://edgeguides.rubyonrails.org/active_record_postgresql.html#uuid